components/image-builder-mk3/cmd/run.go

```diff
@@ -87,52 +56,23 @@ var runCmd = &cobra.Command{
 			go resolver.StartCaching(ctx, interval)
 			service.RefResolver = resolver
 		}
-		if promreg != nil {
-			err = service.RegisterMetrics(promreg)
-			if err != nil {
-				log.Fatal(err)
-			}
-		}
 
-		err = service.Start(ctx)
+		err = service.RegisterMetrics(srv.MetricsRegistry())
 		if err != nil {
-			log.Fatal(err)
+			log.WithError(err).Fatal("Failed to register metrics.")
```

Comment:
nit

```suggestion
			log.WithError(err).Fatal("failed to register metrics.")
```

Reply:
This is the log message which can remain capitalized, the error that it attaches is however lowercased